lib/std/Build/Step/Run.zig

```diff
@@ -240,6 +242,40 @@ pub fn addPrefixedDirectoryArg(self: *Run, prefix: []const u8, directory_source:
     directory_source.addStepDependencies(&self.step);
 }
 
+/// Add a standard `--dep-file=<deps.d>` argument for the child process to fill
+/// with its discovered additional dependencies.
+pub fn addDepFileOutputArg(self: *Run) std.Build.LazyPath {
+    return self.addPrefixedDepFileOutputArg("--dep-file=");
+}
```

Comment:
This should not hard code a "standard" prefix, there is no such standard. The use case for a function named `addDepFileOutputArg` is adding the path to the dep file as a single argument.